components/scream/tests/rrtmgp/CMakeLists.txt

```diff
@@ -1,19 +1,35 @@
 INCLUDE (ScreamUtils)
 
 # Required libraries
-SET (rrtmgpLib rrtmgp)
-SET (NEED_LIBS ${rrtmgpLib} scream_control scream_share)
+set (NEED_LIBS rrtmgp scream_control scream_share yakl)
+
+set (SRC rrtmgp_stand_alone.cpp ${SCREAM_BASE_DIR}/src/physics/rrtmgp/tests/rrtmgp_test_utils.cpp)
 
 # Copy yaml input file to run directory
 CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/input.yaml
                ${CMAKE_CURRENT_BINARY_DIR}/input.yaml COPYONLY)
 
 # Test atmosphere processes
-CreateUnitTest(rrtmgp_stand_alone "rrtmgp_stand_alone.cpp" "${NEED_LIBS}" LABELS "rrtmgp;physics")
+CreateUnitTest(rrtmgp_stand_alone "${SRC}" "${NEED_LIBS}" LABELS "rrtmgp;physics")
+set_source_files_properties("${SRC}" PROPERTIES COMPILE_FLAGS "${YAKL_CXX_FLAGS}")
+target_include_directories(rrtmgp_stand_alone PUBLIC ${SCREAM_BASE_DIR}/../../externals/YAKL)
```

Comment:
Ugh, YAKL really needs to add its src folder to its target include directories, so we don't need to do this...

Reply:
You mean something like this in YAKL's CMakeLists.txt?
```
add_library(yakl STATIC ${YAKL_SOURCE})
target_include_directories(yakl PUBLIC ./)
```
I can make this change upstream if so.

Reply:
Yes, exactly that.